controls/radpivotgrid/features/olap-providers/distinct-values-limit.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Distinct Values Limit
+page_title: Distinct Values Limit
+description: This article shows how to change the distinct values limit in the OlapDataProviders of RadPivotGrid for WPF
```

Comment:
Is this property available only in WPF? If that is the case, we should exclude this article from the SL docs. If not - we can simply omit the "for WPF" part.